xgcm/grid.py

```diff
@@ -1370,15 +1382,21 @@ def get_metric(self, array, axes):
                             [d for mv in possible_combinations for d in mv.dims]
                         )
                         if metric_dims.issubset(array_dims):
-                            # we found a set of metrics with dimensions compatible with the array
+                            # Condition 3: use provided metrics with matching dimensions to calculate for required metric
                             metric_vars = possible_combinations
                             break
-                        # else:
-                        #     # Condition 5: metrics in the wrong position (must interpolate before multiplying)
-                        #     metric_vars = tuple(
-                        #         self.interp_like(pc, array)
-                        #         for pc in possible_combinations
-                        #     )
+                        else:
+                            # Condition 4: metrics in the wrong position (must interpolate before multiplying)
+                            possible_combinations_dims = [
```

Comment:
really minor comments: can we use `possible_dims` instead so it's a single line?